src/Options/OptionsDetails.hpp

```diff
@@ -42,6 +42,42 @@ std::string name() noexcept {
   return name_helper<T>::name();
```

Comment:
I think the main thing that's missing is an update to the tutorial in `docs/DevGuide/OptionParsing.md`.